### PR TITLE
Fix a bunch of stat var descriptions based on gemma eval results

### PR DESCRIPTION
**Pull request by shifucun**
- Add "carbon footprint" to greenhouse  gas emissions description
- Remove `(Non-Biogenic)` in GHG variables to be consistent
- use "without" for No_HealthInsurance var descriptions.

svdiff: https://storage.mtls.cloud.google.com/datcom-embedding-diffs/boxu_base_uae_mem_2024_06_27_15_06_07.html

### CODE REVIEW

**Review by pradh — COMMENTED** [review]
LG.  Maybe we can go over the diffs when we meet?

How does svindex_differ look?

File: tools/nl/embeddings/input/base/sheets_svs.csv [hunk 1]
```diff
@@ -826,47 +826,47 @@ Annual_Emissions_GreenhouseGas_FuelCombustionForRefrigerationAirConditioning,Amo
...
 Annual_Emissions_GreenhouseGas_SteelManufacturing,Amount of Greenhouse Gas Emissions from Steel Manufacturing
 Annual_Emissions_GreenhouseGas_Transportation,Amount of Greenhouse Gas Emissions from Transportation
-Annual_Emissions_GreenhouseGas_UndergroundCoalMines_NonBiogenic,Amount of Greenhouse Gas Emissions from Underground Coal Mines (Non Biogenic)
+Annual_Emissions_GreenhouseGas_UndergroundCoalMines_NonBiogenic,Amount of Greenhouse Gas Emissions from Underground Coal Mines
 Annual_Emissions_GreenhouseGas_WasteManagement,Amount of Greenhouse Gas Emissions from Waste Management
 Annual_Emissions_GreenhouseGas_WastewaterTreatmentAndDischarge,Amount of Greenhouse Gas Emissions from Wastewater Treatment And Discharge
-Annual_Emissions_Hydrofluorocarbon_NonBiogenic,Amount of Hydrofluorocarbon Emissions (Non Biogenic)
-Annual_Emissions_Hydrofluoroether_NonBiogenic,Amount of Hydrofluoroether Emissions (Non Biogenic)
+Annual_Emissions_Hydrofluorocarbon_NonBiogenic,Amount of Non Biogenic Hydrofluorocarbon Emissions
+Annual_Emissions_Hydrofluoroether_NonBiogenic,Amount of Non Biognic Hydrofluoroether Emissions
```
Comment (pradh) [inline tools/nl/embeddings/input/base/sheets_svs.csv:869]: typo

**Review by pradh — APPROVED** [review]
Nice!!!

File: tools/nl/embeddings/input/base/sheets_svs.csv (hunk 1 above)
Comment (shifucun) [reply tools/nl/embeddings/input/base/sheets_svs.csv]: fixed

**Pull request merged**